job_definitions/stats_snapshots.yml

```diff
@@ -1,14 +1,17 @@
+{# frameworks = (framework_slug, enabled?) #}
+{% set frameworks = [('digital-outcomes-and-specialists-4', True)] %}
 {% set environments = ['production'] %}
-{% set frameworks = [('digital-outcomes-and-specialists-4', 'false')] %}
 ---
 {% for environment in environments %}
-{% for framework, disabled in frameworks %}
+{% for framework, enabled in frameworks %}
 - job:
-    name: 'snapshot-stats-{{ framework }}-{{ environment }}'
-    disabled: {{ disabled }}
-    display-name: "Snapshot {{ framework }} stats - {{ environment }}"
-    project-type: freestyle
+    name: stats-snapshot-{{ framework }}-{{ environment }}
+    display-name: Snapshot stats for {{ framework }} - {{ environment }}
     description: Creates an hourly snapshot of framework stats and stores it in the API audit event
+    {% if not enabled %}
+    disabled: true
+    {% endif %}
```

Comment:
I think this should be fine. Though be careful with whitespace/indentation of `{% for %}` loops in Jenkins jobs templates, we've come unstuck with those before 😿

Reply:
Eep good point

Reply:
I'll change it